Test/viper/predicate_exhale.bpl

```diff
@@ -0,0 +1,767 @@
+// RUN: %boogie "%s" | %OutputCheck "%s"
```

Comment:
You used a slightly different way to check Boogie output in the file above. Any particular reasons for using two different ways of checking?

Reply:
Yes, one assertion in predicate_exhale.bpl is expected to fail. I added another "CHECK-L" for that assertion. It can be seen here: https://github.com/boogie-org/boogie/blob/101b776424e22b81b1d3e7e304b34e520b7316ac/Test/viper/predicate_exhale.bpl#L763 

In known_folded.bpl no assertion is supposed to fail. This distinction is the reason for using two different ways of checking.

Thanks a lot for looking into the PR!

Reply:
Sorry, but I am still confused :).
I don't understand the difference between:
```
// RUN: %boogie -proc:known_folded_1 -proc:known_folded_2 "%s" > "%t"
// RUN: %OutputCheck --file-to-check "%t" "%s"
```
and
```
// RUN: %boogie "%s" | %OutputCheck "%s"
```

Don't they accomplish the same thing? If not, they how are they different?
I am just trying to learn myself. Thanks!

Reply:
Oh, I misunderstood your question, sorry.

I must admit that I did not even think about the difference. I just looked at one example each from the repository where no failure is expected and one where a failure is expected. In those two cases they used different ways to run the test. I agree that one should rather do this in a consistent way here.

I think the commands do the same thing in this case, but I am not sure in general. Either way:
```
// RUN: %boogie "%s" | %OutputCheck "%s"
// CHECK-L: Boogie program verifier finished with 2 verified, 0 errors
```
does the expected thing. 

I suggest that I change the way "known_folded.bpl" runs the test to 
// RUN: %boogie "%s" | %OutputCheck "%s"
// CHECK-L: Boogie program verifier finished with 2 verified, 0 errors

That way at least the tests in the Viper folder are run in a consistent way.

Do you agree?

Thanks a lot for the comment!

Reply:
Yes, I agree. That would be good to make uniform, and so please make this update. Then we can merge this in.

Reply:
I have made the update.